src/main/java/edu/ucsb/cs156/gauchoride/entities/RideRequest.java

```diff
@@ -0,0 +1,31 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "riderequests")
+public class RideRequest {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private Long userid;
```

Comment:
Might be good to call this riderUserId to make sure it's distinguished from driverUserId, since eventually we might be assigning drivers to rides as well.